CHANGELOG.md

```diff
@@ -18,13 +18,47 @@ Version 10 of the Sentry JavaScript SDK primarily focuses on upgrading underlyin
 
 Version 10 of the SDK is compatible with Sentry self-hosted versions 24.4.2 or higher (unchanged from v9). Lower versions may continue to work, but may not support all features.
 
+### Init changed for Sentry Vue and Nuxt
+
+Instead of ading the Nuxt/Vue options into Sentry.init options, you will now have to add it inside of `siblingOptions`, this only applies to parameters specific to the respective SDK, other SDKs like React or Angular won't have to do that:
```

Comment:
```suggestion
Instead of adding the Nuxt/Vue options into Sentry.init options, you will now have to add it inside `siblingOptions`, this only applies to parameters specific to the respective SDK, other SDKs like React or Angular won't have to do that:
```